cloudinit/distros/freebsd.py

```diff
@@ -158,4 +168,9 @@ def update_package_sources(self):
             "update-sources", self.package_command,
             ["update"], freq=PER_INSTANCE)
 
+
+class Distro(FreeBSD):
+    pass
```

Comment:
this is a weird construct
i think i understand why it exists, but it's quite weird